tGUI.m

```diff
@@ -52,16 +52,16 @@
 %
 % * Be sure to click Thermoesthesia GUI window with mouse after beginning
 %   to record so that button events move marker and shiver status box.
+% 
+% * When prompted for file name, press Enter to set name to default.
 %
 % Example:
 %
-%   % open figure black background to make Thermoesthesia GUI clearly visible
-%   hf555 = figure(555); set(hf555,'Color',[0 0 0],'menubar','none');
```

Comment:
Is it still possible to open a black background figure before the GUI appears? It's helpful to be able to set this as the screen background when displaying the GUI on a projector.